src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -292,13 +292,13 @@ export const TradeQuote: FC<TradeQuoteProps> = memo(
     ].includes(errors?.[0]?.error)
     const showSwapper = !!quote || showSwapperError
 
-    const totalEstimatedExecutionTimeMs = useMemo(
-      () =>
-        quote?.steps.reduce((acc, step) => {
-          return acc + (step.estimatedExecutionTimeMs ?? 0)
-        }, 0),
-      [quote?.steps],
-    )
+    const totalEstimatedExecutionTimeMs = useMemo(() => {
+      if (quote?.steps.every(step => step.estimatedExecutionTimeMs === undefined)) return
```

Comment:
This was incorrect and was transforming undefined into 0, which have two totally different meaning (instant execution within a Tx vs. unknown)

Reply:
Saner